style/style.css

```
@@ -0,0 +1,7 @@
+.circle {
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  background-color: turquoise;
+  margin: 200px;
```

```suggestion
  margin: 50px;
```

Bele yazsan daha duzgun olar, cunki 200px dizayna uygun gelmir. Bizim $base variable ni islet.